### PR TITLE
cut off header text in modals

### PR DESCRIPTION
**Pull request by eighnjel**
Before:
screen shot 2017-02-02 at 4 34 05 pm

After:
screen shot 2017-02-02 at 4 33 56 pm

### CODE REVIEW

**Comment by eighnjel** [comment]
cc @mischelle-m

**Comment by mischelle-m** [comment]
can we use ellipses at the end when it truncates?

**Comment by eighnjel** [comment]
screen shot 2017-02-02 at 4 41 57 pm

File: Makefile [hunk 1]
```diff
@@ -48,7 +48,7 @@ border-radius-styles:
 	@echo "Generating border-radius style definitions..."
 	@node genBorderRadius.js
 
-LINT_MAX_LESS_PROBLEMS := 189
+LINT_MAX_LESS_PROBLEMS := 190
```
Comment (peternga) [inline Makefile]: Why did it increase?

**Comment by peternga** [comment]
LGTM

**Comment by mischelle-m** [comment]
can we add a little more spacing between the ellipses and the 'x'? it's just a _little_ tight...

**Pull request merged**